package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1-alpha.1",
   "private": true,
   "dependencies": {
-    "@rsksmart/rif-relay-sdk": "0.0.1-alpha.6",
+    "@rsksmart/rif-relay-sdk": "github:rsksmart/rif-relay-client#feature/PP-204",
```

Comment:
we should restore this dependency once we merge the related PRs

Reply:
updated